bindings/nodejs/README.md

```diff
@@ -1,21 +1,21 @@
-# IOTA Wallet Library - Node.js binding
+# IOTA SDK Library - Node.js binding
 
 ## Requirements (only for building the binary yourself)
 
-Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/wallet.rs/blob/dev/README.md) and on Windows also LLVM, our workflow uses `https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`. On Windows, you may also need to set an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
+Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/iota-sdk/tree/develop/sdk#readme) and on Windows also LLVM, our workflow uses `https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`. On Windows, you may also need to set an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
```

Comment:
Is that LLVM link still up-to-date? Asking because there a newer LLVM versions and I don't know if we really require the linked one.

Reply:
It works, no idea if a newer one will also work

Reply:
latest LLVM is now at version 16 (https://github.com/llvm/llvm-project/releases/), so I don't know whether we should investigate upgrading this dependency.

Reply:
Opened an issue https://github.com/iotaledger/iota-sdk/issues/348